packages/shared/lib/services/notification/webhook.service.ts

```diff
@@ -317,6 +321,7 @@ class WebhookService {
 
         const headers = {
             ...nangoHeaders,
+            'Source-Payload-Content-Type': webhookOriginalHeaders['content-type'],
```

Comment:
Not sold on this name, but felt this is useful so that users will know what the providers content type was for the original payload. For providers like slack it isn't json.